components/03-modules/popup/popup.js

```diff
@@ -60,44 +60,6 @@
       this.popup.open = false;
     }
 
-    position() {
-      const buttonCoords = this.button.getBoundingClientRect(),
-            buttonTop    = buttonCoords.height + buttonCoords.top,
-            popupHeight  = this.popup.offsetHeight;
-      if ((window.innerHeight - buttonTop) < popupHeight ) {
-        const popupTop = ((buttonCoords.top - popupHeight) > popupHeight) ? buttonCoords.top - popupHeight : 0;
-        this.popup.style.setProperty('top', `${popupTop}px`);
-      }
-      else {
-        this.popup.style.setProperty('top', `${buttonTop}px`);
-      }
-
-      if (buttonCoords.left > 0 && this.popup.offsetWidth > this.button.offsetWidth) {
-        let leftPos = buttonCoords.left - ((this.popup.offsetWidth - this.button.offsetWidth) / 2);
-        leftPos = (leftPos < 0) ? 0 : leftPos;
-        this.popup.style.setProperty('left', `${leftPos}px`);
-      }
-      else {
-        this.popup.style.setProperty('left', `${buttonCoords.left}px`);
-      }
-
-      this.isOutOfscreen();
-    }
-
-    isOutOfscreen() {
-      const popupContentCoords = this.popup.getBoundingClientRect();
-
-      if (window.matchMedia('(max-width: 480px)').matches) {
-        this.popup.classList.add(this.fullWidth);
-        this.popup.style.setProperty('left', 0);
-      }
-      else if (popupContentCoords.right >= this.body.clientWidth) {
-        this.popup.classList.remove(this.fullWidth);
-        this.popup.style.setProperty('right', 0);
-        this.popup.style.setProperty('left', 'auto');
-      }
-    }
-
```

Comment:
so now we will be set position of each popup in styles?

Reply:
yes